spec/requests/dashboards_spec.rb

```diff
@@ -174,28 +174,24 @@
       expect(response.body).to include 'Your collections'
       expect(response.body).to include '+ Create a new collection'
       expect(response.body).to include <<-HTML
-      <tr>
-        <td><a href=\"#{collection_path(workful_collection)}\">MyString</a></td>
-        <td>5</td>
-        <td>1</td>
-        <td>1</td>
-        <td>1</td>
-        <td>1</td>
-        <td>1</td>
-        <td>Dec 02, 2020</td>
-      </tr>
+  <td><a href=\"#{collection_path(workful_collection)}\">MyString</a><strong><i></i></strong></td>
```

Comment:
These two tests wouldn't pass without spacing these over after moving them into a component. Same below.

Reply:
I feel like checking for exact html in a request spec is a bit of an anti-pattern, but I don't think we need to fix it here as it was a pre-existing condition.